src/pages/wallet-overview/wallet-overview.ts

```diff
@@ -16,89 +19,187 @@ import { ICurrency } from '../../models/ICurrency';
 @Component({
   selector: "page-wallet-overview",
   templateUrl: "wallet-overview.html",
+  animations: [
+    trigger('visibilityChanged', [
+      state('shown', style({ opacity: 1 })),
+      state('hidden', style({ opacity: 0 , display: "none"})),
+      transition('* => *', animate('500ms'))
+    ])
+  ]
 })
 export class WalletOverviewPage {
```

Comment:
A lot of variables have type 'any' in this class. Please double check if there really are no typings available!